src/main/java/TextView.java

```diff
@@ -30,10 +45,17 @@ public void update(Observable o, Object arg)
 	 * abbreviation for a given piece and the outer frame of the board.
 	 * 
 	 * @param board Current board state.
-	 * @throws InvalidPositionException
 	 */
 	void drawBoard(Board board)
 	{
+		int rowCount = 1;
+		System.out.println("     a   b   c   d   e   f   g   h    ");
+		System.out.print(SPACE);
+		
+		for (int i =0; i < board.GRID_SIZE; i++){
+			System.out.print("----");
+		}
+		System.out.println("  ");
 		for (int row = 0; row < board.GRID_SIZE; row++)
 		{
 			for (int col = 0; col < board.GRID_SIZE; col++)
```

Comment:
- [ ] Uppercase the 'b' in `board.GRID_SIZE`, because it is a static variable, so we should referencing the class, not the object.